### PR TITLE
Add ability to associate multiple tasks with a deployment

### PR DESCRIPTION
**Pull request by afgane**
This opens up opportunities for doing status updates and instance termination. 

With the current model migration, past launch task status data is not migrated/preserved. This primarily includes application deployment status and access IP and will be visible for users on their _My appliances_ page as just blank fields.

### CODE REVIEW

File: django-cloudlaunch/baselaunch/tasks.py [hunk 1]
```diff
@@ -1,6 +1,7 @@
 """Tasks to be executed asynchronously (via Celery)."""
 import copy
 import json
+from time import sleep
```
Comment (machristie) [inline django-cloudlaunch/baselaunch/tasks.py]: Probably don't need this import?

File: django-cloudlaunch/baselaunch/urls.py [hunk 2]
```diff
@@ -29,10 +29,16 @@
 # router.register(r'images', views.ImageViewSet)
 router.register(r'infrastructure', views.InfrastructureView,
                 base_name='infrastructure')
-router.register(r'deployments', views.DeploymentViewSet)
+router.register(r'deployments', views.DeploymentViewSet, base_name='deployements')
```
Comment (machristie) [inline django-cloudlaunch/baselaunch/urls.py]: typo: deployements -> deployments

**Comment by afgane** [comment]
Thanks @machristie for taking a look and catching those issues.

File: django-cloudlaunch/baselaunch/views.py [hunk 3]
```diff
@@ -531,21 +531,38 @@ class DeploymentViewSet(viewsets.ModelViewSet):
     """
     List compute related urls.
     """
-    queryset = models.ApplicationDeployment.objects.all()
+    # queryset = models.ApplicationDeployment.objects.all()
```
Comment (nuwang) [inline django-cloudlaunch/baselaunch/views.py]: Should this be uncommented?

Comment (afgane) [inline django-cloudlaunch/baselaunch/views.py]: Just removed it. It get overwritten by the `get_queryset` method below (https://stackoverflow.com/a/29015663)

Comment (nuwang) [inline django-cloudlaunch/baselaunch/views.py]: Right, makes sense.

**Pull request merged**